File: UserModule/src/main/java/com/cap/dto/Train.java
package com.cap.dto;


//import org.springframework.data.mongodb.core.mapping.Document;



import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Train {
	
	

	private Integer  trainNum;
	private String  trainName;
	private String startLoc;
	private String endLoc;
	private Integer seat;
	private String time;
	private double firstAc;
	private double secondAc;
	private double thirdAc;
	private double sleeper;

}

File: Railway-Resercation-System/src/app/admin/train.ts
export class Train{
    constructor(
       public  trainNum:Number,
       public trainName:String,
       public startLoc:String,
       public endLoc:String,
       public  seat:Number,
       public time:String,
       public  firstAc:Number,
       public  secondAc:Number,
       public  thirdAc:Number,
       public sleeper:Number
    ){}
}